packages/client/wallets/smart-wallet/src/blockchain/wallets/service.ts

```diff
@@ -128,6 +129,41 @@ export class SmartWalletService {
                 address: entryPointVersion === "v0.6" ? ENTRYPOINT_ADDRESS_V06 : ENTRYPOINT_ADDRESS_V07,
             },
             kernelVersion,
+            existingSignerConfig: this.getSigner(signers),
         };
     }
+
+    private getSigner(signers: any[]): SignerData | undefined {
+        if (signers.length === 0) {
+            return undefined;
+        }
+
+        if (signers.length > 1) {
+            throw new CrossmintServiceError("Invalid wallet signer configuration. Please contact support");
```

Comment:
Should this error be more specific?

Reply:
It probably could, but will tackle outside this PR since this code was directly moved over from somewhere else!